config/constants.ts

```diff
@@ -1,9 +1,8 @@
 const BASE_PROTOCOL = (!!process.env.IS_LOCAL && 'http') || 'https';
 const BASE_URL = process.env.NEXT_PUBLIC_VERCEL_URL;
-const API_URL = new URL(`${BASE_PROTOCOL}://${BASE_URL}/api/roadmap`);
+export const API_URL = new URL(`${BASE_PROTOCOL}://${BASE_URL}/api/roadmap`);
 
-const TEN_DAYS_IN_SECONDS = 864000;
-const OFFSET_MIN_MONTHS = 2;
-const OFFSET_MAX_MONTHS = 3;
-
-export { API_URL, TEN_DAYS_IN_SECONDS, OFFSET_MIN_MONTHS, OFFSET_MAX_MONTHS };
+export const TEN_DAYS_IN_SECONDS = 864000;
+export const OFFSET_MIN_MONTHS = 2;
+export const OFFSET_MAX_MONTHS = 3;
+export const DEFAULT_TICK_COUNT = 5;
```

Comment:
we should enable eslint rules to enforce these things. I prefer single export, but don't care if it's automated. would prefer reverting this change as it's not necessarily improving anything